timed/tracking/filters.py

```diff
@@ -122,9 +122,11 @@ def filter_editable(self, queryset, name, value):
 
             # only owner, reviewer or supervisor may change reports
```

Comment:
Maybe should update the comment as well to reflect the implemented policy

Reply:
adjusted.